outline/shadowsocks/config.go

```diff
@@ -15,18 +15,25 @@
 package shadowsocks
 
 import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
 	"github.com/Jigsaw-Code/outline-ss-server/client"
 	"github.com/eycorsican/go-tun2socks/common/log"
 )
 
 // Config represents a shadowsocks server configuration.
-// Exported via gobind.
+// Exported via gobind.  Must match ShadowsocksSessionConfig interface in Typescript.
+//
+// TODO: Make this private, so platform code is not exposed to
+// Shadowsocks internals.
 type Config struct {
```

Comment:
Can we mark this as deprecated somehow?

Reply:
Done.  (There is a [lightweight convention](https://go.dev/blog/godoc#:~:text=To%20signal%20that%20an%20identifier%20should%20not%20be%20used%2C%20add%20a%20paragraph%20to%20its%20doc%20comment%20that%20begins%20with%20%E2%80%9CDeprecated%3A%E2%80%9D%20followed%20by%20some%20information%20about%20the%20deprecation.) for this.)